docs/test/replication-controller-details.md

```diff
@@ -0,0 +1,44 @@
+# Replication controller details
+
+## Prerequisites
+Cluster with few deployed applications, where: 
+
+* at least one is valid
+* at least one is invalid (for example with wrong image) 
+
+## Expected result
+Page displays correctly, especially:
+
+* tables with pods and events are available
+* sidebar is available
+* application details are valid and matching data from `kubectl` command
+* all application actions are available
+* warning events have warning icon nearby to their messages
+
+## Specific factors
+
+| Factor                        | 1                               | 2                       | 3              | 4             | 5            | 6               | 7          | 8      | Comment                                                                               |
+|-------------------------------|---------------------------------|-------------------------|----------------|---------------|--------------|-----------------|------------|--------|---------------------------------------------------------------------------------------|
+| Screen size                   | normal                          | mobile                  | narrow         | middle size   | wide         | active resizing |            |        |                                                                                       |
+| Pods count                    | 1                               | odd                     | even           | 20-30         |              |                 |            |        |                                                                                       |
+| Events count                  | 0                               | 0-10                    | more than 10   |               |              |                 |            |        | events number can be changed for example by scaling replication controller pods count |
```

Comment:
> 0 (thats a good case!! but how to test? ) 

With time some of events disappear, but I don't know if it's a rule...

Is there an upper limit of events? How many events are likely to occur for each pod?

Would have to check. 

Problematic thing with this factor is we cannot directly change events count...